scopegen/generate/json.go

```diff
@@ -2,18 +2,54 @@ package generate
 
 import (
 	"encoding/json"
+	"fmt"
 	"io"
 
 	_ "github.com/golang/protobuf/protoc-gen-go/grpc"
 
 	"github.com/gunk/gunk/scopegen/parser"
 )
 
-// JSON generates mapping between service full method names and its OAuth2 scope in JSON format.
-func JSON(w io.Writer, f *parser.File) error {
+// jsonV1 generates mapping between service full method names and its OAuth2 scope in JSON format.
+func jsonV1(w io.Writer, f *parser.File) error {
 	var result = make(map[string][]string, len(f.Methods))
 	for _, method := range f.Methods {
 		result[method.Name] = method.Scopes
 	}
 	return json.NewEncoder(w).Encode(result)
 }
+
+type model struct {
+	Scopes     map[string]string   `json:"scopes"`
+	AuthScopes map[string][]string `json:"auth_scopes"`
+}
+
+// jsonV2 generates mapping between service full method names and its OAuth2 scope in JSON format.
```

Comment:
nit: maybe explain what the difference between jsonV1 and jsonV2 format on the doc here.
summary of what you have on commit message i think it's ok

Reply:
I added a new section in README.md.